src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -197,6 +197,14 @@ int halide_hexagon_remote_power_hvx_on() {
     return 0;
 }
 
+int halide_hexagon_remote_power_hvx_off() {
+    power_ref_count--;
```

Comment:
It could, though this is not the only thing in the runtime that is not thread safe. However, we could make it so fairly easily, and we should. But I'll do that in a separate PR.